src/button/styles.scss

```diff
@@ -147,6 +147,19 @@
     margin-inline-start: awsui.$space-xxs;
   }
 
+  &.variant-inline-link {
+    > .icon-left {
+      inset-inline-start: 0;
```

Comment:
Why is this needed?

Reply:
Is there a reason whey we change it exclusively for the inline variant and not for all button variants with anicon?

Reply:
This is the only variant that is actually affected by this. Normally a button has an outline around it and so the icon is shifting within that outline. If this is applied to all icons in a button then the gaps between the edge of the button and the icon and the gap between the end of the text and the opposite end of the button are not even.